README.md

```diff
@@ -54,16 +54,22 @@ Options:
 	-s SITE, --site=SITE
 		BPC site
 
-	-u, --save_synapse
-		Save output to Synapse
-
+	-mp MAIN_PATIENT, --main_patient=MAIN_PATIENT
+		Main GENIE clinical patient file
+
+	-ms MAIN_SAMPLE, --main_sample=MAIN_SAMPLE
```

Comment:
This is probably a good stopgap solution, but it would be ideal if the parameter was

```
--release 17.2-consortium
```

Then there's code that looks for the clinical main and sample files

Reply:
Since you've done a lot of work already here, lets hold off on that refactor and make it as a future improvement?

Reply:
@thomasyu888 I'm using synapse id since Chelsea prefers synapse id as she mentioned in last week's coordination meeting. I'm happy to tweak this if needed.

Reply:
@Chelsea-Na can you add your thoughts here?

Reply:
I apologize for any confusion. Both approaches have their pros and cons. While the BPC code typically uses a synID as input for other scripts, it's often more convenient to specify the release name directly. Both methods are valid.

However, the optimal solution would be to have a single input parameter to refer to the release, as shown in Tom's example: either "--release 17.2-consortium" or "--synid syn62173551". This approach is preferred because I can't foresee a scenario where we'd need to specify different release versions for the patient and sample files when running case selection.

Reply:
Gotcha, thanks for clarifying. I think we can go ahead with a format like `--release 17.2-consortium` so it's easy to see what consortium release version the code is using. @thomasyu888 It won't take too long to modify this but I'll wait for your go-ahead.